recipes/scalene/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set package_name = "scalene" %}
+{% set version = "v1.5.10" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/0b/9d/54e252b4d573fd65e5301ae75fc56d397d64027f7953e13394ba3692db26/scalene-1.5.10.tar.gz
```

Would suggest following [the `url` from the `example` recipe here]( https://github.com/conda-forge/staged-recipes/blob/85f1d3abc6fcec742b0ab569b103cf30b4f93f8e/recipes/example/meta.yaml#L16 ). This makes it easier for the autoupdate bot to send version update PRs

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```